controllers/storagecluster/quickstart_constants.go

```diff
@@ -111,13 +111,13 @@ spec:
       description: "You can monitor any storage resources managed by OpenShift Container Storage on the Persistent Storage and Object Service dashboards.
 
 
-   Click Home -> Overview to get to the dashboard page, then click on the appropriate tab.
+   Click Storage -> Overview to get to the dashboard page, then click on the appropriate tab.
 
 
-   The Persistent Storage dashboard tab shows the state of OpenShift Container Storage as a whole, as well as the state of any persistent volumes.
+   The Block&File dashboard tab shows the state of OpenShift Container Storage as a whole, as well as the state of any persistent volumes.
```

Comment:
This looks weird, but just to verify: Is this the correct name for this dashboard, e.g. with no spaces?

Reply:
Apparently not. Updated to `Block and File`.